src/js/components/account/user-settings.js

```diff
@@ -24,6 +26,14 @@ export class UserSettings {
           "Change Password"
         );
         $container.append($changePasswordAnchor);
+
+        if (pushpad) {
+          pushpad("status", (isSubscribed) => {
+            if (isSubscribed) {
```

Comment:
shouldn't this conditional be `if (!isSubscribed)` ?

Reply:
Doh! I swapped it when testing. 😸